system-addon/lib/SectionsManager.jsm

```diff
@@ -34,7 +34,10 @@ const BUILT_IN_SECTIONS = {
     title: {id: "header_recommended_by", values: {provider: options.provider_name}},
     disclaimer: {
       text: {id: options.disclaimer_text || "section_disclaimer_topstories"},
-      link: {href: options.disclaimer_link, id: options.disclaimer_linktext || "section_disclaimer_topstories_linktext"},
+      link: {
```

Comment:
Can you add a comment about why the fallback is needed

Reply:
OK, done.